rfd/XXXX-securing-amis.md

```diff
@@ -0,0 +1,225 @@
+---
+authors: Trent Clarke (trent@goteleport.com)
+state: Draft
+---
+
+# RFD 0XXX - Delivering more secure AMIs
+
+# Required Approvers
+* Engineering: @r0mant
+* Security: (@reedloden && @jentfoo)
+* Product: (@xinding33 || @klizhentas)
+
+## What
+
+This RFD discusses structures and processes to increase the security of the
+Amazon EC2 Disk Images (a.k.a AMIs) used to ship Teleport to customers.
+
+## Why
+
+One way our customers consume Teleport is by using a pre-built and  -configured
+AMI to run Teleport in AWS EC2 instances. As a provider of these images we are 
+(at least partially) responsible for the secutity of all ssoftware included in
+the image, not just teleport. 
+
+We should endeavour to not ship vulnerabilities to our clients, even if those
```

Comment:
```suggestion
We should endeavor to not ship vulnerabilities to our clients, even if those
```

Reply:
That's how we spell it here. ;-)

Reply:
No worries, I was not certain but I thought I would recommend anything I noticed

Reply:
I should probably standardise on US spelling for work docs, though. 

Technically, both US and UK spellings are considered valid in Aus (because we know a losing battle when we see one). Unfortunately, whenever I use a US spelling I can't shake the feeling that my Grade 1 teacher is looking over my shoulder, frowning.

Reply:
*standardize

(sorry; as a UK native who has lived in the US/Canada for 6+ years now, I feel _all_ of your pain)